WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -722,7 +722,12 @@ class AbstractPostListViewController: UIViewController, WPContentSyncHelperDeleg
         })
     }
 
+    let loadMoreCounter = LoadMoreCounter()
     func syncHelper(_ syncHelper: WPContentSyncHelper, syncMoreWithSuccess success: ((_ hasMore: Bool) -> Void)?, failure: ((_ error: NSError) -> Void)?) {
+
+        // See https://github.com/wordpress-mobile/WordPress-iOS/issues/6819
+        _ = loadMoreCounter.increment(properties: propertiesForAnalytics())
```

Comment:
You might already be aware, but if you'd like to avoid the use of `_ = ` here, you can add the `@discardableResult` annotation to the method in `LoadMoreCounter.swift`

Not a show-stopper, but it makes it slight nicer for the caller.

Reply:
TIL :)